integration-tests/src/test/java/uk/gov/di/authentication/api/LoginIntegrationTest.java

```diff
@@ -202,42 +202,22 @@ void shouldCallLoginEndpoint6TimesAndReturn400WhenUserIdLockedOut() throws Json.
         headers.put("X-API-Key", FRONTEND_API_KEY);
         headers.put(TXMA_AUDIT_ENCODED_HEADER, ENCODED_DEVICE_INFORMATION);
 
-        var response1 =
-                makeRequest(
-                        Optional.of(new LoginRequest(email, password, JourneyType.SIGN_IN)),
-                        headers,
-                        Map.of());
-        assertThat(response1, hasStatus(401));
-        var response2 =
-                makeRequest(
-                        Optional.of(new LoginRequest(email, password, JourneyType.SIGN_IN)),
-                        headers,
-                        Map.of());
-        assertThat(response2, hasStatus(401));
-        var response3 =
-                makeRequest(
-                        Optional.of(new LoginRequest(email, password, JourneyType.SIGN_IN)),
-                        headers,
-                        Map.of());
-        assertThat(response3, hasStatus(401));
-        var response4 =
-                makeRequest(
-                        Optional.of(new LoginRequest(email, password, JourneyType.SIGN_IN)),
-                        headers,
-                        Map.of());
-        assertThat(response4, hasStatus(401));
-        var response5 =
-                makeRequest(
-                        Optional.of(new LoginRequest(email, password, JourneyType.SIGN_IN)),
-                        headers,
-                        Map.of());
-        assertThat(response5, hasStatus(401));
-        var response6 =
+        var i = 0;
```

Comment:
A fly by comment - Nice refactoring :-) 

A suggestion to make it even neater:
```
var request = new LoginRequest(email, password, JourneyType.SIGN_IN);

for (int i = 0; i < 5; i++) {
  var response = makeRequest(Optional.of(request), headers, Map.of());
  assertThat(response, hasStatus(401));
}

var response = makeRequest(Optional.of(request), headers, Map.of());

assertThat(response, hasStatus(400));
```